src/ServiceControl/Operations/Msmq/MsmqAuditQueueImporter.cs

```diff
@@ -27,12 +27,6 @@ public MsmqAuditQueueImporter(IDocumentStore store, IBuilder builder, IDequeueMe
             this.store = store;
             this.builder = builder;
             enabled = receiver is MsmqDequeueStrategy;
```

Comment:
Since we've looked the throughput to 300 msg/s should we consider removing the batching all together since we don't actually need it? (ie go with the plain vanilla msmq dequeue strategy)